src/app/modules/common/services.js

```diff
@@ -490,6 +518,12 @@ angular.module('BloomLibraryApp.services', ['restangular'])
 			return restangular.withConfig(authService.config()).one('classes/books', id).remove();
 		};
 
+            this.modifyBookField = function(book, field, value) {
+                var rBook = restangular.withConfig(authService.config()).one('classes/books', book.objectId);
+                rBook[field] = value;
+                rBook.put();
+            };
```

Comment:
No. The only place this is used is in the datagrid currently. In that situation, changes locally are all straight UI. Navigating to anywhere with un-updated local changes is going to re-fetch all data from the server anyway.